packages/governance-sdk/src/governance/serialisation.ts

```diff
@@ -111,8 +111,10 @@ import { deserializeBorsh } from '../tools/borsh';
   writer.length += 1;
 
   if (value.type === VoteTypeKind.MultiChoice) {
-    writer.buf.writeUInt16LE(value.choiceCount!, writer.length);
-    writer.length += 2;
+    writer.buf.writeUInt8(value.choiceCount!, writer.length);
```

Comment:
Please update the struct to match the program struct https://github.com/solana-labs/solana-program-library/blob/1892778b9ddb46545f0e7982065889ee7aec1bac/governance/program/src/state/proposal.rs#L89